elements/k-new-post.html

```diff
@@ -3,6 +3,8 @@
 <link rel="import" href="../bower_components/iron-pages/iron-pages.html">
 <link rel="import" href="../bower_components/iron-image/iron-image.html">
 <link rel="import" href="../bower_components/juicy-html/juicy-html.html">
+<link rel="import" href="../bower_components/iron-flex-layout/iron-flex-layout.html">
```

Comment:
The css for `flex4child`, `flex8child` are already defined at https://github.com/PolymerElements/iron-flex-layout/blob/master/classes/iron-flex-layout.html

so instead import the `iron-flex-layout`, you can import `iron-flex-layout/classes/iron-flex-layout.html` instead, 

then just use the CSS classes like `.layout.horizontal `, or `.flex-4`, `.flex-8` right away

Best,
:trollface: